.vscode/settings.json

```diff
@@ -1,7 +1,9 @@
 {
-	"editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
-	"editor.formatOnPaste": false, // required for vs-code-prettier-eslint
-	"editor.formatOnSave": true, // required for vs-code-prettier-eslint
-	"editor.formatOnType": false, // required for vs-code-prettier-eslint
-	"editor.formatOnSaveMode": "file" // required to format on save for vs-code-prettier-eslint
-}
+  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
+  "editor.formatOnPaste": false, // required for vs-code-prettier-eslint
+  "editor.formatOnSave": true, // required for vs-code-prettier-eslint
+  "editor.formatOnType": false, // required for vs-code-prettier-eslint
+  "editor.formatOnSaveMode": "file", // required to format on save for vs-code-prettier-eslint
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "npm.packageManager": "pnpm"
+}
```

Comment:
I'll add linting for this file (tabs were converted to spaces)

Reply:
thanks! hopefully this will prevent [this](https://github.com/huggingface/huggingface.js/pull/55#discussion_r1129223932) from happening again